### ISSUE TITLE
Scrolling between sections doesn't work on an ipad pro with an external magic trackpad

### ISSUE THREAD

**Issue by krajcsbalazs** [alvarotrigo/fullPage.js#3957]
### Description
When I try to scroll between sections on an external magic trackpad which is connected to an ipad pro it doesn't work, it just scrolls the screen and doesn't trigger section change. 

### Link to isolated reproduction with no external CSS / JS
Added a video about it: [https://www.dropbox.com/s/f7gfnpxic9dgmrn/ipad-pro-trackpad.mp4?dl=0](https://www.dropbox.com/s/f7gfnpxic9dgmrn/ipad-pro-trackpad.mp4?dl=0). It does that on the demo page too so it is not a project related issue.

### Steps to reproduce it
Try to scroll on an external magic trackpad which is connected to an Ipad pro.

### Versions 
Ipad pro, Ios 13.4

Is there anything to fix this?

**Comment by vmate** [comment]
Any updates @alvarotrigo?

**Comment by alvarotrigo** [comment]
Nope. Not yet.

**Comment by alvarotrigo** [comment]
I'm not able to reproduce the problem by using iOS in a real ipad device on https://browserstack.com.
And I have no way to connect a trackpad to my iphone, so perhaps you can help me out here?

Can you go to this URL on your ipad and scroll on the white panel with the trackpad? 
What do you see? Can you post a picture of the result?
https://jsfiddle.net/ynw3eLmc/

**Comment by vmate** [comment]
Hey Alvaro! Unofrtunately I was unable to scroll that panel.. https://share.icloud.com/photos/07T3isGI_kP5iUDiHu3CSFxEg

**Comment by alvarotrigo** [comment]
Yeah it doesn't seem to be firing the mouse wheel event.

I assume your trackpad is a device like this?
![image](https://user-images.githubusercontent.com/1706326/86599620-29c98780-bf9f-11ea-94b8-df1acb89dd15.png)

**Comment by vmate** [comment]
No, it is a Magic keyboard: https://www.apple.com/shop/product/MXQT2LL/A/magic-keyboard-for-ipad-pro-11%E2%80%91inch-2nd-generation-us-english

**Comment by Leofic** [comment]
Hey, I actually have the same issue. It does scroll to the next section if i hit the space bar however.  It is only the track pad that doesn't move anything.
